posconvert.module

```diff
@@ -236,7 +268,12 @@ function posconvert_form($form, &$form_state) {
         $lines = $file_row_count;
       }
 
-      if ($cols < 2 || count($check_rows) <= 0) {
+      if ($lines > 19500) {
+        // Too many lines nearing the allowed line limit.
+        $form['ajax_wrapper']['error']['#markup'] =
+          '<div class="messages error">Text data is near the limit of allowed number of line. Please try again.</div>';
```

Comment:
How about: "Your file exceeds the maximum number of lines. Please try a smaller file or contact us."